jobs/async-upload/samples/create_model_example.yaml

```
@@ -0,0 +1,118 @@
+      volumes:
+      - name: model-metadata
+        configMap:
+          name: model-metadata-configmap
+      - name: s3-credentials
+        secret:
+          secretName: s3-credentials-secret
+      - name: oci-credentials
+        secret:
+          secretName: oci-credentials-secret
+
+      restartPolicy: OnFailure
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)